packages/editor/src/store/actions.js

```diff
@@ -401,10 +307,9 @@ export function createUndoLevel() {
 }
 
 /**
- * Returns an action object used to lock the editor.
+ * Action that locks the editor.
```

Comment:
How about just
```suggestion
 * Locks the editor.
```
without "Action that"?

Reply:
We use "Action that ..." quite universally when documenting actions, there are almost 100 usages of that. Therefore I'd rather keep it as is.